media/js/base/sentry.es6.js

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const Sentry = require('@sentry/browser');
```

Comment:
Just a general question here, since this is using es6 - is there a big difference between using the es6 import syntax? `import Sentry from '@sentry/browser` vs the require?

Reply:
I mostly kept this using `require()` for consistency, since we support commonJS in all scripts, and not just `es6.js` scripts. Happy to switch it if you think it makes more sense though, thoughts?

Reply:
That's possible using `require()` as well I think. But we can just go ahead and use the newer syntax anyway I think. I'll make it so 👍